File: Trip-Planner/src/app/classes/Plaats.ts
import { Activiteit } from "./Activiteit";
import { Gebruiker } from "./Gebruiker";
import { Land } from "./Land";

export class Plaats {
    plaatsId: number;
    land: Land;
    naam:string;
    postcode: string;
    gebruikers: Gebruiker[] = [];
    activiteiten: Activiteit[] = [];

    constructor(plaatsId: number, land: Land, naam:string, postcode: string,) {
        this.plaatsId = plaatsId;
        this.land = land;
        this.naam = naam;
        this.postcode = postcode;
    }
}

File: TripPlannerAPI/TripPlannerDAL/Entity/Plaats.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripPlannerDAL.Entity
{
    public class Plaats
    {
        [Key]
        public int plaatsId { get; set; }
        public int? landId { get; set; }
        public Land? land { get; set; }
        public string naam { get; set; }
        public string postcode { get; set; }
        public ICollection<Gebruiker>? gebruikers { get; set; }
        public ICollection<Activiteit>? activiteiten { get; set; }
    }
}
